201-vm-domain-join-existing/README.md

```diff
@@ -0,0 +1,9 @@
+# Create a new VM on a new storage account from a custom image
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-vm-domain-join-existing%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template allows you to join a Windows Virtual Machine into an Active Directory Domain.
+
+Important to notice that the DomainJoinUserName parameter follows the domain\username format, where domain is the NetBios Domain Name.
```

Comment:
As well, put more detail in here to describe what users need to do to get this deployment to succeed - since it relies on a pre-existing configuration (AD created, VM created, user with perms to join domain)